File: 28_copy_string.c
// C program to copy one string into another and count number of characters occupied


#include <stdio.h>
#include<string.h>

int main() {
    char original[100], copy[100];      //declared a character array of 100 capacity
    int length;
    
    printf("Enter any string: ");
    gets(original);
    
    length=strlen(original);
    
    //copy string character by character
    for(int i=0; i<length; i++){
        copy[i]=original[i];
    }
    
    printf("\nCopied string is: %s", copy);
    printf("\nLength of string is: %d",length);

    return 0;
}

File: 28_copy_string.py
# Python program to copy one string into another

#design function to copy one string into another
def copy_string(original):
    """Copy one string into another."""
    copied_str = original[:]
    return copied_str

def main():
    # Get the original string from the user
    original_str = input("Enter the original string: ")

    # Copy the string
    copied_str = copy_string(original_str)

    # Display the result
    print(f"Original String is: {original_str}")
    print(f"Copied String is: {copied_str}")

if __name__ == "__main__":
    main()
